app/uk/gov/hmrc/eoricommoncomponent/frontend/controllers/EoriLanguageController.scala

```diff
@@ -27,11 +27,7 @@ class EoriLanguageController @Inject() (config: Configuration, languageUtils: La
     extends LanguageController(config, languageUtils, cc) {
 
   override protected def fallbackURL: String =
-    "/customs-enrolment-services/register" //This will be always register for cds we might need to add a route for fallback cannot be dynamic
-
-  // TODO this is used just in tests and lang argument is not used in this method
-  def langToCall(lang: String): String => Call =
-    EoriLanguageController.routeToSwitchLanguage
+    "/customs-enrolment-services/subscribe" //This will be always register for cds we might need to add a route for fallback cannot be dynamic
```

Comment:
Thinking about it, this also won't work without the service in the url.   Can you add a small TODO to look at later.   In general this should not be a problem if the language switcher always works.

Reply:
Yeah, I'll add TODO